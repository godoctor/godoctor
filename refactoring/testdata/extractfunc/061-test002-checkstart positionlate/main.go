//<<<<<extract,9,4,11,33,foo,pass
package main

import "fmt"

func main() {
	b := 4
	c := 5
	b = 5
	c = 6
	fmt.Println("IN EXTRACT", b, c)
	fmt.Println("IN MAIN", b, c)
}
