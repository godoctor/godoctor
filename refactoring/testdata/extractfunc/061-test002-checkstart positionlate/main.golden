//<<<<<extract,9,4,11,33,foo,pass
package main

import "fmt"

func main() {
	b := 4
	c := 5
	b, c = foo()
	fmt.Println("IN MAIN", b, c)
}
func foo() (int, int) {
	var b int
	var c int
	b = 5
	c = 6
	fmt.Println("IN EXTRACT", b, c)
	return b, c
}
